src/main.js

```diff
@@ -334,13 +331,16 @@ function getConfig(request) {
   
   config.newInfo()
   .setId('Caching Credentials')
-  .setText('You have set credentials to be cached: ' +  user.getProperty('dscc.areCredentialsCached'));
+  .setText('Currently, your credentials are [' + (user.getProperty('dscc.areCredentialsCached') == 'true' ? 'being cached' : 'not being cached') + ']. If you ' +
```

Comment:
GDS unfortunately does not allow a lot of text configuration so I cannot bold statements. The idea here is to tell the user whether they are currently caching their password or not. This choice is the default if they do not select a new choice.